packages/utils/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# 1.0.0
```

Comment:
Please remove this file, versioning doesn't make any sense for this package. We can add it in case this library will be published to npm

Reply:
This will be published to npm definitely soon as we are going to implement this in trezor-connect.

Reply:
Are you sure? There is no need to publish it if we want to use it, it will be just bundled together during build, same like it will happen now for `rollout` for example.

Reply:
I think I am sure. We want to get rid of these plain.js (flowtype) utils https://github.com/trezor/connect/tree/develop/src/js/utils and replace them with their typescript counterpart. Once trezor-connect is inside trezor-suite monorepo, we won't need npm anymore, thats true.

Reply:
Oh we are have not correcly configured TS using references and composite project and without it, it won't be bundled together that is quite bummer :(

Reply:
I not sure if this event won't break `rollout` package... probably yes

Reply:
I meant `blockchain-link` instead of `rollout`.

Anyway I will try to do at least some hotfix until TS will be globally correctly configured.

Reply:
False alarm, blockchain link uses webpack so it will take care of it... we can resolve this